MobiFlight/BoardDefinitions.cs

```diff
@@ -33,6 +33,18 @@ public static Board GetBoardByUsbVolumeLabel(String volumeLabel)
             return boards.Find(board => board.UsbDriveSettings?.VolumeLabel == volumeLabel);
         }
 
+        /// <summary>
+        /// Finds board definitions by matching against USB drive volume label. This does not check for the
+        /// presence of a secondary file on the drive to confirm it is a supported USB drive.
+        /// This is used for offering different community board options.
+        /// </summary>
+        /// <param name="volumeLabel">The volume label to match</param>
+        /// <returns>The first board definition matching the volumeLabel, or null if none found.</returns>
```

Comment:
This needs to be updated to say it returns all board definitions matching the volumeLabel.